### ISSUE TITLE
Question on Table 4 in the paper

### ISSUE THREAD

**Issue by abhishekaich27** [jeffreychou777/LOTVS-MMAU#5]
@jeffreychou777 Are your results in Table 4 obtained after fine-tuning on training set or zero-shot setting (no fine-tuning on train set) for all methods? Also, can you provide the script used to obtain those values?

**Comment by jeffreychou777** [comment]
The models in Table 4 were trained on the original code settings of their codebase(Except for SeViLA), and are not the result of fine-tuning after pre-training.
Some of the models are too large and only the batchsize was changed.

**Comment by abhishekaich27** [comment]
Was each method pre-trained on a different dataset and then evaluated on the MM-AU val/test set?
